ecs/daemon/inputs/task.conf

```diff
@@ -0,0 +1,16 @@
+[INPUT]
+    Name                tail
+    Tag                 app.*
+    Path                /var/lib/docker/containers/*/*.log
+    Docker_Mode         On
+    Docker_Mode_Flush   5
+    Docker_Mode_Parser  container_firstline
+    Parser              docker
```

Comment:
As I note here, this is the old parser config, we can/should use the new one which supports docker: https://github.com/aws-samples/amazon-cloudwatch-container-insights/pull/115#discussion_r1062927891